boa_engine/src/syntax/parser/expression/left_hand_side/call.rs

```diff
@@ -97,6 +97,25 @@ where
                         TokenKind::Keyword((kw, _)) => {
                             lhs = GetConstField::new(lhs, kw.to_sym(interner)).into();
                         }
+                        TokenKind::BooleanLiteral(bool) => {
+                            match bool {
+                                true => {
+                                    lhs = GetConstField::new(lhs, Keyword::True.to_sym(interner))
+                                        .into();
+                                }
+                                false => {
+                                    lhs = GetConstField::new(lhs, Keyword::False.to_sym(interner))
+                                        .into();
+                                }
+                            };
```

Comment:
Isn't it easier to use `if bool {` here? Having a match over a boolean instead of an if...else seems a bit weird. Also, I'm surprised that we need to do `Keyword::True.to_sym(interner)` instead of just using `Sym::TRUE`. `true`/`false` are pretty used strings, we might want to add them as constants. As a suggestion on the implementation:

```suggestion
                            lhs = GetConstField::new(lhs, if bool { Keyword::True } else { Keyword::False }.to_sym(interner)).into();
```

This probably needs a `cargo fmt`, though.